changelog/v1.18.0-rc3/issue_7309.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
```

Comment:
Our changelog-bot only allows a single file per PR. We can merge these two into a single file, with multiple entries

Reply:
Resolved by commit https://github.com/solo-io/gloo/commit/281ace000966aa7f15d960ab73d60efc4fa1c189